dissect/target/plugins/os/windows/everything/everything_types.py

```diff
@@ -0,0 +1,452 @@
+        self.filesystems_cstruct = filesystems_cstruct(self.version)
+        self.__parse_filesystems()
+
+        # This might be hidden/system files/folders, or maybe a list of folders to exclude?
+        # TODO - Check what happens when changing exclude hidden/system files/folders
+        if self.version > COMPAT_1:
+            exclude_flags = c_header.uint8_t(self.fh)
+            log.debug("Exclude flags: %s", exclude_flags)
+
+        # There is some logic here, but in my test file it doesn't affect the data
+        # (Tested by dumping the raw array while debugging in IDA)
+        # In practice, none of the data in the function is saved, it just advances the offset
+        # This *MIGHT* be affected by an old version of the db (before 1.7.8)
+        # If any of these fail, then I'd have to reverse this function and add support
+
+        # I expect 0 here because the inner implementation of the function is:
+        # for i in range(read_varint(self.fh)):
+        #   do something
+        # and the function is called 3 times one after another. As long as zero is returned each time, we don't need
+        # to implement the logic
+        if [read_varint(self.fh), read_varint(self.fh), read_varint(self.fh)] != [0, 0, 0]:
+            raise NotImplementedError("Failed to parse database, unimplemented feature. Please open an issue")
+
+        # Keep current position in the file, so we can seek back to it during __iter__
+        self._start_seek = self.fh.tell()
+
+    def __repr__(self) -> str:
+        return f"{self.header} - Filesystems{self.filesystem_list}"
+
+    def __parse_filesystems(self) -> None:
+        self.filesystem_list = []
+
+        for i in range(self.header.number_of_filesystems):
+            filesystem_header = self.filesystems_cstruct.filesystem_header(self.fh)
+            log.debug("Filesystem %d: type %s", i, filesystem_header)
+
+            try:
+                fs_type = EverythingFSType(filesystem_header.type)
+            except ValueError:
+                raise ValueError(f"Unknown FS type {filesystem_header.type}")
+
+            if fs_type in [EverythingFSType.NTFS, EverythingFSType.REFS]:
+                if fs_type == EverythingFSType.NTFS:
+                    header = self.filesystems_cstruct.ntfs_header(self.fh)
+                else:
+                    header = self.filesystems_cstruct.refs_header(self.fh)
+
+            elif fs_type == EverythingFSType.EFU:
+                header = self.filesystems_cstruct.efu_header(self.fh)
+
+            elif fs_type == EverythingFSType.FOLDER:
+                header = self.filesystems_cstruct.folder_header(self.fh)
+            else:
+                raise NotImplementedError(f"Have not implemented parsing {fs_type}")
+            self.filesystem_list.append((fs_type, filesystem_header, header))
+
+    def __iter__(self) -> Iterator[EverythingF]:
+        self.fh.seek(self._start_seek, io.SEEK_SET)
```

Comment:
```suggestion
        self.fh.seek(self._start_seek)
```
`SEEK_SET` is already the default.